Update docs for version 0.9.9 (#278)
AequilibraE/qaequilibrae: docs/source/getting_started/getting_started.rst

Step--by-step installation
~~~~~~~~~~~~~~~~~~~~~~~~~~

The steps for installing AequilibraE are the same as for any QGIS plugin.
Go to the Plugins panel and click on **Manage and Install Plugins**.

.. image:: ../images/install_1.png
    :width: 1181
    :align: center
    :alt: First step

.. image:: ../images/install_2.png
    :width: 1035
    :align: center
    :alt: Second step

.. image:: ../images/install_3.png
    :width: 1226
    :align: center
    :alt: Third step

After installing the plugin, you will be faced with the question of whether you
want to download its dependencies, which are required for using most of the

> features. The problem is that many of AequilibraE's algorithms rely on compiled

Reviewer: ```suggestion
features. This is necessary because AequilibraE's algorithms rely on compiled
```
